### PR TITLE
feat: Display latest posts on start and add bluesky social

### PR DESCRIPTION
**Pull request by load1n9**

### CODE REVIEW

File: pages/index.tsx [hunk 1]
```diff
@@ -5,6 +5,7 @@ import { href, output } from "utils/path.ts";
 import { write } from "utils/fs.ts";
 
 import { Layout } from "components/Layout.tsx";
+import { BlogPreviewList } from "../components/BlogPreviewList.tsx";
```
Comment (eliassjogreen) [inline pages/index.tsx]:
```suggestion
import { BlogPreviewList } from "components/BlogPreviewList.tsx";
```
**nitpick:** Absolute import

File: pages/index.tsx [hunk 2]
```diff
@@ -42,6 +43,10 @@ function Index() {
         <a href="https://discord.gg/bwY4TRB8J7">Discord server</a>{" "}
         where the core team resides and where we discuss development.
       </p>
+      <section>
+        <h1>Latest Blog Posts</h1>
```
Comment (eliassjogreen) [inline pages/index.tsx]:
```suggestion
        <h2>Latest Blog Posts</h2>
```
**suggestion:** Probably makes more sense semantically?

File: components/Footer.tsx [hunk 3]
```diff
@@ -19,6 +20,9 @@ export function Footer() {
         <a aria-label="Discord" href="https://discord.gg/bwY4TRB8J7">
           <DiscordIcon />
         </a>
+        <a aria-label="BlueSkye" href="https://bsky.app/profile/trynova.dev">
```
Comment (eliassjogreen) [inline components/Footer.tsx]:
```suggestion
        <a aria-label="Bluesky" href="https://bsky.app/profile/trynova.dev">
```
**issue:** typo lol

File: components/BlogPreviewList.tsx [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+  );
+
+export function BlogPreviewList() {
+  return (
+    <ul class={classes.list}>
+      {allPosts.slice(0, 5).map((post) => (
+        <div class={classes.post}>
+          <a href={href(post.file)}>{post.meta.title}</a>
+          <time datetime={post.meta.date.toString()} class={classes.meta}>
+            {post.meta.date.toLocaleString("sv-SE")}
```
Comment (eliassjogreen) [inline components/BlogPreviewList.tsx:18]: **praise:** The superior swedish date-format

**Review by eliassjogreen — CHANGES_REQUESTED** [review]
Looks good! Just some very minor comments

File: components/BlogPreviewList.tsx (hunk 4 above)
Comment (load1n9) [reply components/BlogPreviewList.tsx:18]: lmao

**Review by eliassjogreen — APPROVED** [review]

**Pull request merged**